mkt/developers/views.py

```diff
@@ -396,7 +399,7 @@ def content_ratings_edit(request, addon_id, addon):
             # No IARC request exists. Create.
             iarc_request = IARCRequest.objects.create(
                 app=addon, uuid=uuid.uuid4())
-        ctx['iarc_request_id'] = iarc_request.uuid
+        ctx['iarc_request_id'] = unicode(uuid.UUID(iarc_request.uuid))
```

Comment:
This is to make sure we're properly formatting the UUID before sending it to IARC through the questionnaire.